data/org.freedesktop.portal.GlobalShortcuts.xml

```diff
@@ -0,0 +1,184 @@
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @options: Unused so far
+
+        List all the shortcuts with their invocation descriptions.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcuts a(sa{sv})</term>
+            <listitem><para>
+              The different shortcut ids that have been registered. They are mapped to a list of pairs including:
+                - description: User-readable text describing what the shortcut does
+                - trigger: User-readable text describing how to trigger the shortcut
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="ListShortcuts">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="o" name="request_handle" direction="out"/>
+    </method>
+
+    <!--
+        BindShortcuts:
+        @session_handle: Object path for the #org.freedesktop.impl.portal.Session object representing the session
+        @shortcuts: The identifier of the shortcuts we intend to register, empty for all shortcuts
+        @request_handle: Object path for the #org.freedesktop.portal.Request object representing this call
+
+        Shows a dialog with all shortcuts to configure and offers them back to the app when done.
+
+        The following results get returned via the #org.freedesktop.portal.Request::Response signal:
+        <variablelist>
+          <varlistentry>
+            <term>shortcuts a(sa{sv})</term>
+            <listitem><para>
+              The different shortcut ids that have been registered. They are mapped to a list of pairs including:
+                - description: User-readable text describing what the shortcut does
+                - trigger: User-readable text describing how to trigger the shortcut
+            </para></listitem>
+          </varlistentry>
+        </variablelist>
+    -->
+    <method name="BindShortcuts">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="as" name="shortcuts" direction="in"/>
+      <arg type="s" name="parent_window" direction="in"/>
+      <arg type="a{sv}" name="options" direction="in"/>
+      <arg type="o" name="request_handle" direction="out"/>
+
+      <annotation name="org.qtproject.QtDBus.QtTypeName.In3" value="QVariantMap"/>
+    </method>
+
+    <!--
+        Activated:
+        Notifies about a shortcut becoming active.
+
+        @session_handle: Session that requested the shortcut
+        @shortcut_id: the application-provided ID for the notification
+        @timestamp: When the event happened in ms
+    -->
+    <signal name="Activated">
+      <arg type="o" name="session_handle" direction="in"/>
+      <arg type="s" name="shortcut_id" direction="in"/>
+      <arg type="t" name="timestamp" direction="in"/>
```

Comment:
Should we specify what clock the timestamp is in? Without it, it's rather useless other than comparing to other `Activated` signal emissions.

Reply:
The unused options argument is on now.

Changed the timestamp to map the wording in the location portal `@timestamp: The timestamp, as seconds and microseconds since the Unix epoch.`